gobblin-completeness/src/main/java/org/apache/gobblin/completeness/verifier/KafkaAuditCountVerifier.java

```diff
@@ -171,16 +171,19 @@ private double calculateClassicCompleteness(String datasetName, long beginInMill
     for (String refTier: this.refTiers) {
       long refCount = countsByTier.get(refTier);
       long srcCount = countsByTier.get(this.srcTier);
+      double tmpPercent;
 
       /*
         If we have a case where an audit map is returned, however, one of the source tiers on another fabric is 0,
         and the reference tiers from Kafka is reported to be 0, we can say that this hour is complete.
         This needs to be added as a non-zero double value divided by 0 is infinity, but 0 divided by 0 is NaN.
        */
       if (srcCount == 0 && refCount == 0) {
-        return 1.0;
+        tmpPercent = 1;
+      } else {
+        tmpPercent =  (double) srcCount / (double) refCount;
       }
-      percent = Double.max(percent, (double) srcCount / (double) refCount);
+      percent = percent < 0 ? tmpPercent : Double.min(percent, tmpPercent);
```

Comment:
If the srcTiers are local cluster 1, local cluster 2, etc, rather than using the lowest percentage completion, would instead it be srcTier / (summation of refTiers)? Since the total records across all the refTiers should be equal to the number of records on srcTier eventually?

Depending on what the eventual conclusion of this is, let's also update the javadocs of this to match the behavior

Reply:
[It's not necessarily for local tier, but in general to support multiple ](https://github.com/apache/gobblin/pull/3806#discussion_r1375129804)

Reply:
https://github.com/apache/gobblin/pull/3806#discussion_r1375129804